pages/common/ssh-copy-id.md

```diff
@@ -0,0 +1,11 @@
+# ssh-copy-id
+
+> Install your public key in a remote machine's authorized_keys.
+
+- Copy your key to a remot machine:
+
+`ssh-copy-id {{user@remote}}`
+
+- Copy a specific key to a remot machine:
+
+`ssh-copy-id -i {{~/.ssh/id_example.pub}} {{user@remote}}`
```

Comment:
Do you think it'd be worth changing the token to "remote_host", for consistency with [ssh.md](https://github.com/tldr-pages/tldr/blob/master/pages/common/ssh.md), [scp.md](https://github.com/tldr-pages/tldr/blob/master/pages/common/scp.md), and probably others?